nav2_msgs/action/AssistedTeleop.action

```diff
@@ -0,0 +1,8 @@
+#goal definition
+builtin_interfaces/Duration time
+---
+#result definition
+builtin_interfaces/Duration total_elapsed_time
+---
+#feedback
+builtin_interfaces/Duration time_left
```

Comment:
remove this, its not really necessary, we know the time left based on the request.